winml/lib/Api/LearningModelSession.cpp

```diff
@@ -361,7 +351,6 @@ LearningModelSession::Evaluate(
     winml::LearningModelBinding binding,
     hstring const& correlation_id) try {
   ToggleProfiler();
-  _winmlt::PerformanceTelemetryEvent kEvaluateModel_event(WinMLRuntimePerf::kEvaluateModel);
 
```

Comment:
I think it would be useful to still have a ETW tracelogging write event that indicates that an evaluate in WinML layer for debugging purposes. I don't think this should be removed. This event doesn't send up telemetry